addon/mixins/lazy-image.js

```diff
@@ -1,15 +1,26 @@
 import Ember from 'ember';
 import Cache from '../lib/cache';
 
-const { on, get, set, Mixin, computed } = Ember;
+const { on, get, set, Mixin, computed, setProperties } = Ember;
 const dasherize = Ember.String.dasherize;
 
 export default Mixin.create({
+  viewportOptionsOveride: on('didInsertElement', function() {
```

Comment:
I don't think so because it shouldn't trigger template re-render? worth looking into